sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveTableScanSuite.scala

```diff
@@ -90,4 +90,11 @@ class HiveTableScanSuite extends HiveComparisonTest {
     assert(sql("select CaseSensitiveColName from spark_4959_2").head() === Row("hi"))
     assert(sql("select casesensitivecolname from spark_4959_2").head() === Row("hi"))
   }
+
+  test("SPARK-8003 Resolve spark__partition__id in Hive queries") {
+    TestHive.sql("CREATE TABLE tb (KEY INT, VALUE STRING)")
```

Comment:
I created the table to emulate the way the rest of the Hive query test were setup, of create table, scan, and drop.